pages/api/table-cell.md

```diff
@@ -22,9 +22,9 @@ import TableCell from '@material-ui/core/TableCell';
 | <span class="prop-name">children</span> | <span class="prop-type">node</span> |   | The table cell contents. |
 | <span class="prop-name">classes</span> | <span class="prop-type">object</span> |   | Override or extend the styles applied to the component. See [CSS API](#css) below for more details. |
 | <span class="prop-name">component</span> | <span class="prop-type">element type</span> |   | The component used for the root node. Either a string to use a DOM element or a component. |
-| <span class="prop-name">padding</span> | <span class="prop-type">enum:&nbsp;'normal'&nbsp;&#124;<br>&nbsp;'checkbox'&nbsp;&#124;<br>&nbsp;'none'<br></span> |   | Sets the padding applied to the cell. By default, the Table parent component set the value. |
+| <span class="prop-name">padding</span> | <span class="prop-type">enum:&nbsp;'default'&nbsp;&#124;<br>&nbsp;'checkbox'&nbsp;&#124;<br>&nbsp;'none'<br></span> |   | Sets the padding applied to the cell. By default, the Table parent component set the value. |
 | <span class="prop-name">scope</span> | <span class="prop-type">string</span> |   | Set scope attribute. |
-| <span class="prop-name">size</span> | <span class="prop-type">enum:&nbsp;'normal'&nbsp;&#124;<br>&nbsp;'small'<br></span> |   | Specify the size of the cell. By default, the Table parent component set the value. |
+| <span class="prop-name">size</span> | <span class="prop-type">enum:&nbsp;'normal'&nbsp;&#124;<br>&nbsp;'small'<br></span> |   | Specify the size of the cell. By default, the Table parent component set the value (`normal`). |
```

Comment:
`default`, now that you undid the change…

Reply:
Oops. The fewer breaking changes we can make in a single pull request, the better. I have reverted the change because it wasn't the main value proposition of this pull request. I think that it would be great to come back to it, with a global approach.

Reply:
We are doing `size?: 'small' | 'medium' | 'large';` for the button and the fab.

Reply:
I have updated the property to be consistent.